dev/com.ibm.ws.security.authorization.jacc.testprovider/bnd.bnd

```diff
@@ -59,4 +59,5 @@ Include-Resource: \
   com.ibm.ws.logging;version=latest,\
   com.ibm.ws.org.osgi.annotation.versioning;version=latest,\
   com.ibm.ws.security.authorization;version=latest,\
+  io.openliberty.jakarta.authorization.2.1,\
```

Comment:
It doesn't appear that you use it in this project.  Don't see a reason to add it to the build path.